include/z64.h

```diff
@@ -37,6 +37,8 @@
 #include "z64object.h"
 #include "z64ocarina.h"
 #include "z64player.h"
+#include "z64scene.h"
+#include "z64schedule.h"
 #include "z64save.h"
 #include "z64scene.h"
```

Comment:
```suggestion
#include "z64schedule.h"
#include "z64save.h"
#include "z64scene.h"
```
z64scene.h seems to have snuck in there twice, likely due to git merging